src/plugins/discover/public/plugin.tsx

```diff
@@ -390,9 +390,11 @@ export class DiscoverPlugin
     });
   };
 
-  private registerEmbeddable(core: CoreSetup<DiscoverStartPlugins>, plugins: DiscoverSetupPlugins) {
-    const ebtManager = new DiscoverEBTManager(); // It is not initialized outside of Discover
-
+  private registerEmbeddable(
+    core: CoreSetup<DiscoverStartPlugins>,
+    plugins: DiscoverSetupPlugins,
+    ebtManager: DiscoverEBTManager
+  ) {
```

Comment:
This change enables sending EBT events for embeddables: `discover_field_usage` and `discover_profile_resolved`. I hope that's fine.